package.json

```diff
@@ -4,8 +4,7 @@
   "description": "In-app purchase validation for Apple, Google, Amazon, Roku",
   "main": "index.js",
   "scripts": {
-    "test": "eslint .",
-    "precommit": "npm test"
+    "test": "eslint . --fix"
```

Comment:
I'm not sure if I would want to autofix precommit, since you wouldn't be able to review the changes when ESLint makes them, until you make a pull request. Interesting thought though it is, maybe just remove the `--fix`?

Reply:
Yes, you're right. I will remove the `--fix`. Maybe is better to do check the errors and fix them manually before committing.